association-frontend/components/section-social-media/section-social-media.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import DiscordIcon from "../icons/discord";
+import FacebookIcon from "../icons/facebook";
+import GitHubIcon from "../icons/github";
+import InstagramIcon from "../icons/instagram";
+import LinkedInIcon from "../icons/linkedin";
+import MailIcon from "../icons/mail";
+import TwitterIcon from "../icons/twitter";
+import SectionItem from "../section-item/section-item";
+import SocialCard from "../social-card/social-card";
+
+const SectionSocialMedia = () => {
+  const icons = [
+    <DiscordIcon />,
+    <FacebookIcon />,
+    <GitHubIcon />,
+    <InstagramIcon />,
+    <LinkedInIcon />,
+    <TwitterIcon />,
+    <MailIcon />,
+  ];
+  return (
+    <SectionItem title={"Seguici Online 🎭"}>
+      <div className="flex flex-row space-x-4 w-full  justify-center">
+        {icons.map((icon) => {
+          return (
+            <div className="flex-shrink-0">
+              {" "}
+              <SocialCard component={icon} />
+            </div>
+          );
+        })}
```

Comment:
```suggestion
        {icons.map((icon, index) => {
          return (
            <div key={index} className="flex-shrink-0">
              {" "}
              <SocialCard component={icon} />
            </div>
          );
        })}
```

when rendering from an array you need to specify a `key` otherwise react complains it cannot keep track of the objects, using the index isn't ideal but in this case should be fine